src/main/java/org/tosca/docs/model/InterfaceMethod.java

```diff
@@ -0,0 +1,27 @@
+package org.tosca.docs.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.tosca.docs.model.impl.InterfaceImpl;
+import org.tosca.docs.model.impl.InterfaceMethodImpl;
+
+/**
+ * Created by prabvenu on 3/31/17.
+ */
+@JsonDeserialize(as = InterfaceMethodImpl.class)
+public interface InterfaceMethod extends  Comparable<InterfaceMethod>{
+
+
+     String getName();
```

Comment:
Please make sure the "reformat code", "rearrange code" and other goodies are enabled in your IntelliJ commit dialog.
Please run it on the changed classes.